frontend/src/core/codemirror/format.ts

```diff
@@ -58,3 +63,37 @@ export function formatAll(updateCellCode: CellActions["updateCellCode"]) {
   const views = notebookCellEditorViews(getNotebook());
   return formatEditorViews(views, updateCellCode);
 }
+
+export function canToggleMarkdown(editorView: EditorView | null) {
+  if (
+    !editorView ||
+    editorView.state.field(languageAdapterState).type === "markdown"
+  ) {
+    return false;
+  }
+  return (
+    LanguageAdapters.markdown().isSupported(
+      getEditorCodeAsPython(editorView),
+    ) || getEditorCodeAsPython(editorView).trim() === ""
+  );
+}
+
+export function toggleToMarkdown(
+  cellId: CellId,
+  editorView: EditorView,
+  updateCellCode: CellActions["updateCellCode"],
+) {
+  if (!canToggleMarkdown(editorView)) {
+    return;
+  }
+  if (getEditorCodeAsPython(editorView).trim() === "") {
+    const blankMd = 'mo.md("")';
```

Comment:
What do you think about defaulting to a raw f-string? That will make it possible to interpolate Python values with `{}`, and make it easier to write LaTex. ie

```
mo.md(
    rf"""
    """
)
```

Reply:
Done, I don't think the triple quotes is required since it seems to get removed to `rf""` whenever I switch back and forth, but adding actual python seems to automatically trigger going to triple quotes?